com.unity.multiplayer.mlapi/Runtime/Profiling/ProfilerCountersInfo.cs

```diff
@@ -75,6 +75,9 @@ private static void RegisterMLAPIPerformanceEvent()
 
         private static void OnPerformanceTickData(PerformanceTickData tickData)
         {
+            if(tickData == null)
```

Comment:
Maybe add the description of this PR as a code comment here so that we later remember why this null check exists?

Reply:
I commented that inline, I think it looks better now.

Reply:
Changed this to just prevent null data from escaping